aiodocker/containers.py

```diff
@@ -75,10 +75,10 @@ def __init__(self, docker):
         try:
             await container.start()
         except DockerError as err:
-            raise DockerError(
+            raise DockerContainerError(
                 err.status,
                 {"message": err.message},
-                container_id=container['id']
+                container['id']
                 )
```

Comment:
Please de-indent this closing parenthesis!

Reply:
closed